### PR TITLE
Initialize shader uniforms by default

### LINKED ISSUE #44454: Shader uniforms has null as default value
**Godot version:**

3.2.4 beta4

**Steps to reproduce:**
1. Create a 2D node with shader material
2. Paste this code
```
shader_type canvas_item;

uniform mat4 test;
```
3. Save the scene
4. Open the tscn in a text editor
5.
```
[sub_resource type="ShaderMaterial" id=2]
shader = SubResource( 1 )
shader_param/test = null
```

### PR DESCRIPTION
**Pull request by Chaosus**
Fix #44454

### CODE REVIEW

**Review by KoBeWi — COMMENTED** [review]
Seems to work although I can't test my original case on master.

I wonder if it's possible to not save default values in the scene (like with regular properties).

File: servers/rendering/renderer_rd/renderer_storage_rd.cpp [hunk 1]
```diff
@@ -1453,7 +1453,19 @@ void RendererStorageRD::shader_get_param_list(RID p_shader, List<PropertyInfo> *
 	Shader *shader = shader_owner.getornull(p_shader);
 	ERR_FAIL_COND(!shader);
 	if (shader->data) {
-		return shader->data->get_param_list(p_param_list);
+		shader->data->get_param_list(p_param_list);
+
+		// assign default values
```
Comment (KoBeWi) [inline servers/rendering/renderer_rd/renderer_storage_rd.cpp:1458]:
```suggestion
		// Assign default values.
```

Comment (Chaosus) [reply servers/rendering/renderer_rd/renderer_storage_rd.cpp]: Fixed

**Comment by KoBeWi** [comment]
I tested again and it doesn't fix the issue. idk, maybe something changed since my original comment.

**Comment by Chaosus** [comment]
Not sure why it's not working, I close it for now.

**Pull request closed without merging**